src/test/java/org/springframework/data/couchbase/repository/N1qlCrudRepositoryIntegrationTests.java

```diff
@@ -152,7 +153,12 @@ public void shouldFailConversionWithStringReturnType() {
   public void shouldDoNumericProjectionWithStringBasedQuery() {
     Party partyHasKeyword = new Party(KEY_PARTY_KEYWORD, "party", "desc is a N1QL keyword", new Date(), 4000000, new Point(500, 500));
     partyRepository.save(partyHasKeyword);
-
+    Optional<Party> p = null;
+    int count = 0;
+    do {
+       try { Thread.sleep(1000); } catch(Exception e){}
+       try { p = partyRepository.findById(partyHasKeyword.getKey()); } catch (DataRetrievalFailureException e) {}
+    } while ( !p.isPresent() && count++ < 60);
     long max = partyRepository.findMaxAttendees();
```

Comment:
wouldn't it be easier to use scan consistency on findMaxAttendees so that request plus is  implied?